src/libs/SidebarUtils.js

```diff
@@ -236,7 +240,8 @@ function getOptionData(reportID) {
     result.isChatRoom = ReportUtils.isChatRoom(report);
     result.isArchivedRoom = ReportUtils.isArchivedRoom(report);
     result.isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(report);
-    result.shouldShowSubscript = result.isPolicyExpenseChat && !report.isOwnPolicyExpenseChat && !result.isArchivedRoom;
+    result.isMoneyRequestReport = ReportUtils.isMoneyRequestReport(report);
+    result.shouldShowSubscript = (result.isPolicyExpenseChat && !report.isOwnPolicyExpenseChat && !result.isArchivedRoom) || ReportUtils.isIOUReport(report);
```

Comment:
This should be expense reports not for IOU right? @luacmartins @Julesssss 
```suggestion
    result.shouldShowSubscript = (result.isPolicyExpenseChat && !report.isOwnPolicyExpenseChat && !result.isArchivedRoom) || ReportUtils.isExpenseReport(report);
```